test/behaviour/graql/GraqlSteps.java

```diff
@@ -113,13 +113,17 @@ public void graql_undefine_throws_exception(String exception, String undefineQue
     }
 
     @Given("graql insert; throws exception")
+    @SuppressWarnings("ReturnValueIgnored")
     public void graql_insert_throws(String insertQueryStatements) {
-        assertThrows(() -> graql_insert(insertQueryStatements).iterator().next());
+        //noinspection ResultOfMethodCallIgnored
+        assertThrows(() -> graql_insert(insertQueryStatements).collect(Collectors.toList()));
```

Comment:
This change addresses some flaky tests where `graql insert throws` step was failing on random tests.